liqd_product/assets/scss/components/_headings.scss

```diff
@@ -3,3 +3,7 @@
     text-transform: uppercase;
     letter-spacing: 0.1em;
 }
+
+.heading--sm-margin {
```

Comment:
Sorry, still no modifier! I would have called it heading-sm-margin. But I will merge anyway!